src/SFA.DAS.AssessorService.Application.Api/Services/CertificateNameCapitalisationService.cs

```diff
@@ -0,0 +1,199 @@
+            NonEnglishFamilyNameReplacements.Add(@"\bDell([ae])\b", "dell$1");          // della and delle Italian.
+            NonEnglishFamilyNameReplacements.Add(@"\bD([aeiou])(?!-)\b", "d$1");        // da, de, di Italian; du French; do Brasil
+            NonEnglishFamilyNameReplacements.Add(@"\bD([ao]s)\b", "d$1");               // das, dos Brasileiros
+            NonEnglishFamilyNameReplacements.Add(@"\bDe([lrn])(?=\s+\w)", "de$1");      // del Italian; der/den Dutch/Flemish or forename Del.
+            NonEnglishFamilyNameReplacements.Add(@"\bEl\b", "el");                      // el Greek or El Spanish.
+            NonEnglishFamilyNameReplacements.Add(@"\bLa\b", "la");                      // la French or La Spanish.
+            NonEnglishFamilyNameReplacements.Add(@"\bPar La Grâce\b", "par la grâce");  // par la grâce French.
+            NonEnglishFamilyNameReplacements.Add(@"\bL([eo])\b", "l$1");                // lo Italian; le French.
+            NonEnglishFamilyNameReplacements.Add(@"\bVan(?=\s+\w)", "van");             // van German or forename Van.
+            NonEnglishFamilyNameReplacements.Add(@"\bVon\b", "von");                    // von Dutch/Flemish
+            NonEnglishFamilyNameReplacements.Add(@"\bThe\b", "the");                    // The to the
+            NonEnglishFamilyNameReplacements.Add(@"\bOf\b", "of");                      // Of to of
+            NonEnglishFamilyNameReplacements.Add(@"\bAnd\b", "and");                    // And to and
+            NonEnglishFamilyNameReplacements.Add(@"\bSon\b", "son");                    // Son to son
+        }
+
+        /// <summary>
+        /// Replace characters in a name part with colloquial upper or lower case letters
+        /// </summary>
+        /// <param name="namePart">A part of name e.g. Given Name or Family Name</param>
+        /// <param name="familyNamePart">Indicates whether special rules apply for a Family Name</param>
+        /// <returns></returns>
+        public string ProperCase(string namePart, bool familyNamePart = false)
+        {
+            if (string.IsNullOrEmpty(namePart))
+                return namePart;
+
+            CleanseStringForSpecialCharacters(ref namePart);
+            Capitalize(ref namePart);
+
+            if (familyNamePart)
+            {
+                UpdateMac(ref namePart);
+
+                foreach (KeyValuePair<string, string> replacement in NonEnglishFamilyNameReplacements)
+                {
+                    string pattern = replacement.Key;
+                    namePart = Regex.Replace(namePart, pattern, replacement.Value, RegexOptions.IgnoreCase);
+                }
+
+                FixConjunction(ref namePart);
+            }
+
+            return namePart;
+        }
+
+        /// <summary>
+        /// Cleans all magic characters
+        /// </summary>
+        /// <returns></returns>
+        public void CleanseStringForSpecialCharacters(ref string inputString)
```

Comment:
Check if these methods are used and make private if they are not and remove from interface